### PR TITLE
Update fIX_bOOK_cONTROL.java

### PR DESCRIPTION
**Pull request by chandrika-katikam**

### CODE REVIEW

File: src/library/fixbook/fIX_bOOK_cONTROL.java [hunk 1]
```diff
@@ -18,11 +18,11 @@ public fIX_bOOK_cONTROL() {
 	}
 	
 	
-	public void SeT_Ui(FixBookUI ui) {
+	public void SeT_uI(FixBookUI uI) {
```
Comment (ttawan) [inline src/library/fixbook/fIX_bOOK_cONTROL.java:21]: Change to ```setUI``` not SeT_UI

Comment (ttawan) [reply src/library/fixbook/fIX_bOOK_cONTROL.java:21]: ```public void setUI(FixBookUI UI)```

File: src/library/fixbook/fIX_bOOK_cONTROL.java [hunk 2]
```diff
@@ -18,11 +18,11 @@ public fIX_bOOK_cONTROL() {
 	}
 	
 	
-	public void SeT_Ui(FixBookUI ui) {
+	public void SeT_uI(FixBookUI uI) {
 		if (!StAtE.equals(CoNtRoL_StAtE.INITIALISED)) 
 			throw new RuntimeException("FixBookControl: cannot call setUI except in INITIALISED state");
 			
-		this.Ui = ui;
+		this.Ui = uI;
```
Comment (ttawan) [inline src/library/fixbook/fIX_bOOK_cONTROL.java:25]: ```this.UI = UI```

**Comment by parneetbatra** [comment]
all good

**Review by aatish-rajkarnikar — APPROVED** [review]

**Comment by parneetbatra** [comment]
all good

**Review by parneetbatra — APPROVED** [review]

File: src/library/fixbook/fIX_bOOK_cONTROL.java (hunk 1 above)
Comment (ttawan) [inline src/library/fixbook/fIX_bOOK_cONTROL.java:21]: Change to ```set_UI```

File: src/library/fixbook/fIX_bOOK_cONTROL.java (hunk 2 above)
Comment (ttawan) [inline src/library/fixbook/fIX_bOOK_cONTROL.java:25]: Change to ```this.UI```

**Review by ttawan — CHANGES_REQUESTED** [review]

**Pull request merged**